### ISSUE TITLE
Ember 2.16.2 Deprecation warning Ember.String.singularize()

### ISSUE THREAD

**Issue by eharrow** [funkensturm/ember-local-storage#271]
```
DEPRECATION: Ember.String.singularize() is deprecated. Please explicitly: import { singularize } from 'ember-inflector'; [deprecation id: ember-inflector.globals]
        at logDeprecationStackTrace (http://localhost:4200/assets/vendor.js:29324:21)
        at HANDLERS.(anonymous function) (http://localhost:4200/assets/vendor.js:29546:9)
        at raiseOnDeprecation (http://localhost:4200/assets/vendor.js:29354:14)
        at HANDLERS.(anonymous function) (http://localhost:4200/assets/vendor.js:29546:9)
        at invoke (http://localhost:4200/assets/vendor.js:29558:9)
        at Object.deprecate (http://localhost:4200/assets/vendor.js:29412:24)
        at Object.get (http://localhost:4200/assets/vendor.js:114427:13)
        at Module.callback (http://localhost:4200/assets/vendor.js:116715:34)
        at Module.exports (http://localhost:4200/assets/vendor.js:111:32)
```

in /node_modules/ember-local-storage/addon/helpers/import-export.js

Thanks

**Comment by fsmanuel** [comment]
@eharrow Thanks for reporting. Do you mind opening a PR to fix it? I'll be happy to merge.

**Comment by artzte** [comment]
Thanks for this awesome add-on! +1 on clearing this deprecation warning.

**Comment by fsmanuel** [comment]
@eharrow @artzte @mminkoff  released as `1.4.1`

**Comment by fsmanuel** [comment]
@jlblcc  released as `1.4.1`
